src/main/java/com/example/car_dealership/models/Dealerships.java

```diff
@@ -0,0 +1,17 @@
+package com.example.car_dealership.models;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Dealerships {
+    private int dealership_id;
```

Comment:
nit: Even though the column in SQL is snake case, the property in Java should still be camel case, e.g. dealershipId